Source/CardUtils.swift

```diff
@@ -1,65 +1,19 @@
-    /// Dictionary of card types based on their scheme
-    let cardTypes: KeyValuePairs<CardScheme, CardType> = [
-        .visa: CardType(scheme: .visa,
-                        name: "Visa",
-                        pattern: "^4\\d*$",
-                        gaps: [4, 8, 12],
-                        validLengths: [16],
-                        validCvvLengths: [3],
-                        isLuhnChecked: true),
-        .mastercard: CardType(scheme: .mastercard,
-                              name: "Mastercard",
-                              pattern: "^(5[1-5]|222[1-9]|22[3-9]|2[3-6]|27[0-1]|2720)\\d*$",
-                              gaps: [4, 8, 12],
-                              validLengths: [16],
-                              validCvvLengths: [3],
-                              isLuhnChecked: true),
-        .americanExpress: CardType(scheme: .amex,
-                                   name: "American Express",
-                                   pattern: "^3[47]\\d*$",
-                                   gaps: [4, 10],
-                                   validLengths: [15],
-                                   validCvvLengths: [4],
-                                   isLuhnChecked: true),
-        .dinersClub: CardType(scheme: .diners,
-                              name: "Diner's Club",
-                              pattern: "^3(0[0-5]|[689])\\d*$",
-                              gaps: [4, 10],
-                              validLengths: [14, 16, 19],
-                              validCvvLengths: [3],
-                              isLuhnChecked: true),
-        .discover: CardType(scheme: .discover,
-                            name: "Discover",
-                            pattern: "^(6011|65|64[4-9])\\d*$",
-                            gaps: [4, 8, 12],
-                            validLengths: [16, 19],
-                            validCvvLengths: [3],
-                            isLuhnChecked: true),
-        .jcb: CardType(scheme: .jcb,
-                       name: "JCB",
-                       pattern: "^(2131|1800|35)\\d*$",
-                       gaps: [4, 8, 12],
-                       validLengths: [16, 17, 18, 19],
-                       validCvvLengths: [3],
-                       isLuhnChecked: true),
-        .maestro: CardType(scheme: .maestro,
-                           name: "Maestro",
-                           // swiftlint: disable line_length
-                           pattern: "^(?:5[06789]\\d\\d|(?!6011[0234])(?!60117[4789])(?!60118[6789])(?!60119)(?!64[456789])(?!65)6\\d{3})\\d{8,15}$",
-                           // swiftlint: enable line_length_violation
-                           gaps: [4, 8, 12],
-                           validLengths: [12, 13, 14, 15, 16, 17, 18, 19],
-                           validCvvLengths: [3],
-                           isLuhnChecked: true)
-    ]
+    let cardGaps: [Card.Scheme: [Int]] = [.visa: [4, 8, 12],
```

Comment:
Is this part of checkout iOS SDK ?

Reply:
No

Reply:
Task : https://checkout.atlassian.net/browse/PIMOB-1193